src/coreComponents/common/format/table/unitTests/testTable.cpp

```diff
@@ -289,41 +247,84 @@ TEST( testTable, layoutTable )
              );
 }
 
-TEST( testTable, tableSetMargin )
+TEST( testTable, subColumns )
+{
+  {
+    TableLayout tableLayout( " ",
+                             "Column1",
+                             TableLayout::ColumnParam{"Nodes ", TableLayout::Alignment::right, true, {"Locales", "Ghost", "Active"}},
+                             "Column3",
+                             TableLayout::ColumnParam{"Column4 ", TableLayout::Alignment::right, true, {"Locales", "Ghost"}},
+                             "Column5" );
+
+    TableData tableData;
+    tableData.addRow( "min", "125", "375,0001", " YES", 2354654, 562, 43.0, 43.0, 562, 5 );
+    tableData.addRow( "max", "360", "390,1", " YES", 383213213, 712, 48.0, 47.0, 72, 2 );
+
+    TableTextFormatter tableLog( tableLayout );
+
+    EXPECT_EQ( tableLog.toString( tableData ),
+               "\n--------------------------------------------------------------------------------------------------------\n"
+               "|       |  Column1  |                            Nodes   |  Column3  |           Column4   |  Column5  |\n"
+               "--------------------------------------------------------------------------------------------------------\n"
+               "|       |           |   Locales  |  Ghost  |     Active  |           |  Locales  |  Ghost  |           |\n"
+               "--------------------------------------------------------------------------------------------------------\n"
```

Comment:
Header texts should be centered or justified at the left side, but cannot be justified on the right side (general standard).
I have a preference to centering it since the title is centered by default. Can you follow the same behaviour the Title has?